addon/ng2/models/webpack-build-common.ts

```diff
@@ -0,0 +1,92 @@
+import * as webpack from 'webpack';
+import {LoaderConfig} from '../utilities/ts-path-mappings-webpack-plugin';
+
+const path = require('path');
+const ForkCheckerPlugin = require('awesome-typescript-loader').ForkCheckerPlugin;
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+export const getWebpackCommonConfig = function(projectRoot: string) {
```

Comment:
`export function getWebpackCommonConfig(projectRoot: string) {` instead of setting an anonymous function. I know it's the same, but doesn't read the same ;)